src/components/PlioQuestion.vue

```diff
@@ -160,6 +176,7 @@ export default {
       if (this.plioQuestion.state == "answered") {
         setTimeout(() => {
           // highlight wrong/right depending on what the user answered in previous session
+          document.getElementById("options_container").classList.add("options-block");
```

Comment:
This is done because earlier the `display` of the options container was `block` but then when the tutorial message was added (`choose only one`), the `display` had to be changed to `flex` to incorporate that message as well.
BUT, when the tutorial message is not required, the `display` needs to be switched back to `block` otherwise the css breaks and it looks weird

Reply:
oh boy. I will take your word for it. But usually, with CSS, there's a way around hacks. Maybe if we use `grid`? We can look later!